packages/sedlex/sedlex.2.1/opam

```diff
@@ -23,12 +23,15 @@ build: [
 ]
 depends: [
   "ocaml" {>= "4.02.3"}
-  "dune" {>= "1.0"}
+  "dune" {>= "1.8"}
   "ppx_tools_versioned"
   "ocaml-migrate-parsetree"
   "gen"
   "uchar"
 ]
+conflicts: [
```

Comment:
why does this introduce two unrelated conflicts?

Reply:
I would rather see those be modified in their own packages rather than this one if it's just about dune

Reply:
Ok!